libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -386,7 +393,7 @@ void AP_GPS_SBF::broadcast_configuration_failure_reason(void) const
 bool AP_GPS_SBF::is_configured (void) {
     return (!gps._raw_data || (RxState & SBF_DISK_ACTIVITY)) &&
             (gps._auto_config != AP_GPS::GPS_AUTO_CONFIG_DISABLE &&
-             _init_blob_index < ARRAY_SIZE(_initialisation_blob));
+             _init_blob_index >= ARRAY_SIZE(_initialisation_blob));
```

Comment:
Squash this fix (and just the fix, the rest of the commit is fine as separate).

Reply:
Whoops, totally missed the comment for some reason...